Aptoide/aptoide-client-v8
# [ASV-1797] Fix cobrand my account

**What does this PR do?**

   This PR aims at changing the string of my account in cobrand, related with the Aptoide products section.
Also took the chance to extract the variable related with the version of OKHTTP and update it.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] MyAccountFragment.java

**How should this be manually tested?**

  Run the app in both vanilla and cobrand flavours and check that everything is compiling ok. 
Also, since we updated okhttp 

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1797](https://aptoide.atlassian.net/browse/ASV-1797)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
